src/app/Shared/Services/NotificationChannel.service.tsx

```diff
@@ -105,8 +109,17 @@ export class NotificationChannel {
         },
         (err: any) => this.logError('Notifications URL configuration', err)
       );
+
+    combineLatest(this.login.loggedOut())
```

Comment:
This can be unwrapped/the `combineLatest` removed, since there is only one inner observable.